src/ansys/units/dimensions.py

```diff
@@ -1,52 +1,39 @@
-"""Provides the ``Dimensions`` and ``BaseDimensions`` class."""
-from enum import Enum
+"""Provides the ``Dimensions`` class."""
+import ansys.units as ansunits
 
 
-class BaseDimensions(Enum):
+class Dimensions:
     """
-    Supplies all valid base dimensions used in dimensional analysis.
+    A class which contains the base unit information.
```

Comment:
@ansaboutin This comment is a bit inaccurate; i.e.: _unit information_ is what the `Unit`/`Units` class would be for. Also, note that the comment does not begin _A class which..._ because that's redundant.  See [here](https://numpydoc.readthedocs.io/en/latest/format.html) for good examples.